cmd/nri-prometheus/config.go

```diff
@@ -4,25 +4,46 @@ package main
 
 import (
 	"fmt"
+	"path/filepath"
 	"reflect"
 	"regexp"
 	"strings"
 	"time"
 
+	"github.com/newrelic/infra-integrations-sdk/args"
 	"github.com/newrelic/nri-prometheus/internal/cmd/scraper"
 	"github.com/pkg/errors"
 	"github.com/spf13/viper"
 )
 
+// ArgumentList Available Arguments
+type ArgumentList struct {
+	ConfigPath string `default:"test" help:"Path to the config file"`
```

Comment:
Can't we use viper to read the flag?

Reply:
Yes but in accepting args we can build a similar experience to other integration, es:
 - `help` command with documentation
 - we should add common commands, es `version`
Standard errors:
```
flag provided but not defined: -config_path2
Usage of nri-prometheus:
  -config_path string
        Path to the config file (default "test")
```